components/appHeader/HeaderProfileDropdown.vue

```diff
@@ -30,16 +36,26 @@
 
 
 <script setup lang="ts">
+const { locale, locales } = useI18n();
+const localePath = useLocalePath();
+const switchLocalePath = useSwitchLocalePath()
+
 const userStore = useUserStore()
 const isProfileMenuOpen = ref(false)
 const profileMenuRef = ref<HTMLElement | null>(null)
 const loggedIn = ref(false)
 
 
+const availableLocales = computed(() => {
+  return locales.value.filter(i => i.code !== locale.value)
+})
+
+
 async function logout() {
   await userStore.logout()
   isProfileMenuOpen.value = true
-  navigateTo("/register")
+  localStorage.clear()
```

Comment:
if the user is signed in annonymously should't this work differently

Reply:
The user cannot access this menu when they are signed in anonymously